### PR TITLE
PRAC-11-Registration-form-frontend

### PR DESCRIPTION
**Pull request by ElenaDu**
## One Line Description
UI with basic functionality for Login/Registration Form
## Requirements
See Figma for design details 

## Notes

Forms verification and additional functionality will be added in the next Sprint
## Test Steps

endpoint http://localhost:5173/login

## UI/UX Outcome

Screen Shot 2024-03-17 at 6 26 25 PM
Screen Shot 2024-03-17 at 6 26 07 PM

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -17,7 +19,7 @@ const App = () => {
             <Routes>
                 <Route path="/" element={<Layout />}>
                     {/* <Route index element={<HomePage />} /> */}
-                    {/* <Route path="/login" element={<LoginForm />} /> */}
+                    <Route path="/login" element={<AuthPage activeTab={'login'} />} />
```
Comment (IrynaKolh) [inline src/App.tsx]: maybe better to write path="auth" because it is common for login and registration

**Review by IrynaKolh — APPROVED** [review]

File: src/App.tsx (hunk 1 above)
Comment (ElenaDu) [reply src/App.tsx]: Thank you! I was thinking the same, and I'll make this change in the next pull request, if that's alright.

Comment (IrynaKolh) [reply src/App.tsx]: It is small changes, better to make them now. For this exist 2 resons: 1) you mostly complete auth functionality and to open a new PR for changing route it is irrational. 2) we need navigate to auth route from many components, and wait for next PR for changing it is too long. Also when i write code now I renember about wrong route, but 1 week late I can forget.

**Review by oxangyal — APPROVED** [review]
Looks good, but would change the route the sooner then later:)

**Comment by ElenaDu** [comment]
Fixed!

**Pull request merged**